bundles/binding/org.openhab.binding.satel/src/main/java/org/openhab/binding/satel/config/IntegraStateBindingConfig.java

```diff
@@ -0,0 +1,258 @@
+/**
+ * Copyright (c) 2010-2014, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.satel.config;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.openhab.binding.satel.SatelBindingConfig;
+import org.openhab.binding.satel.internal.event.IntegraStateEvent;
+import org.openhab.binding.satel.internal.event.SatelEvent;
+import org.openhab.binding.satel.internal.protocol.SatelMessage;
+import org.openhab.binding.satel.internal.protocol.command.ControlObjectCommand;
+import org.openhab.binding.satel.internal.protocol.command.IntegraStateCommand;
+import org.openhab.binding.satel.internal.types.DoorsState;
+import org.openhab.binding.satel.internal.types.InputState;
+import org.openhab.binding.satel.internal.types.IntegraType;
+import org.openhab.binding.satel.internal.types.ObjectType;
+import org.openhab.binding.satel.internal.types.OutputControl;
+import org.openhab.binding.satel.internal.types.OutputState;
+import org.openhab.binding.satel.internal.types.StateType;
+import org.openhab.binding.satel.internal.types.ZoneControl;
+import org.openhab.binding.satel.internal.types.ZoneState;
+import org.openhab.core.items.Item;
+import org.openhab.core.library.items.ContactItem;
+import org.openhab.core.library.items.NumberItem;
+import org.openhab.core.library.items.SwitchItem;
+import org.openhab.core.library.types.DecimalType;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.library.types.OpenClosedType;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.State;
+import org.openhab.model.item.binding.BindingConfigParseException;
+
+/**
+ * This class implements binding configuration for all items that represents
+ * Integra zones/inputs/outputs state.
+ * 
+ * @author Krzysztof Goworek
+ * @since 1.7.0
+ */
+public class IntegraStateBindingConfig implements SatelBindingConfig {
+
+	private static final DecimalType DECIMAL_ONE = new DecimalType(1);
+
+	private StateType stateType;
+	private int objectNumber;
+	private Map<String, String> options;
```

Comment:
They are private, it's not so common to add javadoc to private stuff, at least as far as I know. I can add some non javadoc comment if you wish.